projects/plugins/jetpack/modules/videopress/class.jetpack-videopress.php

```diff
@@ -58,6 +58,37 @@ public function on_init() {
 
 		VideoPress_Scheduler::init();
 		VideoPress_XMLRPC::init();
+
+		if ( $this->is_videopress_enabled() ) {
+			add_action( 'admin_notices', array( $this, 'media_new_page_admin_notice' ) );
+		}
+	}
+
+	/**
+	 * The media-new.php page isn't supported for uploading to VideoPress.
+	 *
+	 * There is either a technical reason for this (bulk uploader isn't overridable),
+	 * or it is an intentional way to give site owners an option for uploading videos that bypass VideoPress.
+	 */
+	public function media_new_page_admin_notice() {
+		global $pagenow;
+
+		if ( 'media-new.php' === $pagenow ) {
+			echo '<div class="notice notice-warning is-dismissible">' .
+					'<p>' .
+					wp_kses(
+						sprintf(
+							/* translators: %s is the url to the Media Library */
+							__( 'VideoPress uploads are not supported here. To upload to VideoPress, add your videos from the <a href="%s">Media Library</a>.', 'jetpack' ),
+							'/wp-admin/upload.php'
```

Comment:
It may be safer to use `admin_url()` here?

```suggestion
							esc_url( admin_url( 'upload.php' ) )
```